api/src/main/resources/rdr.yaml

```diff
@@ -0,0 +1,202 @@
+#This file provide the end points for interaction with RDR
+
+
+swagger: '2.0'
+
+info:
+  title: RDR
+  description: |
+    Service for RDR Interactions.
+  version: "0.1"
+#The host will get updated as per the environment it will be pmi-drc-api-test for dev and test
+host: "pmi-drc-api-test.appspot.com"
+schemes:
+  - "https"
+basePath: /rdr/v1
+
+securityDefinitions:
+  googleoauth:
+    type: oauth2
+    authorizationUrl: 'https://accounts.google.com/o/oauth2/auth'
+    flow: implicit
+    scopes:
+      openid: open id authorization
+      email: email authorization
+      profile: profile authorization
+
+security:
+  - googleoauth:
+    - openid
+    - email
+    - profile
+    - https://www.googleapis.com/auth/devstorage.full_control
+
+produces:
+  - application/json
+
+##########################################################################################
+## PATHS
+##########################################################################################
+
+paths:
+  /workbench/directory/workspaces:
+    post:
+      description: Submits one or more workspace snapshots for storage in the RDR.
+      tags:
+        - RDR
+      consumes:
+        - application/json
+      operationId: sendWorkspace
```

Comment:
[Non-opt] For operation name here, I'd consider something like "saveWorkspaces" or "updateWorkspaces". Think about if this API were published as part of the Workbench system, we'd probably use words like "save" or "update" rather than "send".

I would also be sure to make the operation use plural wording, since it accepts one or more workspaces.

This comment applies equally to the researchers endpoint.